influxdb_client_3/write_client/client/write/dataframe_serializer.py

```diff
@@ -192,19 +200,21 @@ def __init__(self, data_frame, point_settings, precision=DEFAULT_WRITE_PRECISION
             # field column has no nulls, we don't run the comma-removal
             # regexp substitution step.
             sep = '' if len(field_indexes) == 0 else ','
+
             if issubclass(value.type, np.integer):
-                field_value = f"{sep}{key_format}={{{val_format}}}i"
-            elif issubclass(value.type, np.bool_):
-                field_value = f'{sep}{key_format}={{{val_format}}}'
-            elif issubclass(value.type, np.floating):
                 if null_columns.iloc[index]:
-                    field_value = f"""{{"" if math.isnan({val_format}) else f"{sep}{key_format}={{{val_format}}}"}}"""
+                    field_value = f"""{{"" if pd.isna({val_format}) else f"{sep}{key_format}={{{val_format}}}i"}}"""
+                else:
+                    field_value = f"{sep}{key_format}={{{val_format}}}i"
```

Comment:
This can be simplify to:

```python
 if issubclass(value.type, np.integer) or issubclass(value.type, np.floating) or issubclass(value.type, np.bool_):  # noqa: E501
                suffix = 'i' if issubclass(value.type, np.integer) else ''
                if null_columns.iloc[index]:
                    field_value = f"""{{"" if pd.isna({val_format}) else f"{sep}{key_format}={{{val_format}}}{suffix}"}}"""  # noqa: E501
                else:
                    field_value = f"{sep}{key_format}={{{val_format}}}{suffix}"
```

Reply:
done